sopel/bot.py

```diff
@@ -594,27 +594,26 @@ def rate_limit_info(
     ) -> tuple[bool, Optional[str]]:
         if trigger.admin or rule.is_unblockable():
             return False, None
-
+        nick = trigger.nick
```

Comment:
Little confused why no blank line after the `if`; that's our usual style.

("Applying suggestions on deleted lines is not supported", so sorry, no quick-fix suggestion.)